pennylane/gradients/finite_difference.py

```diff
@@ -243,7 +244,7 @@ def _finite_diff_new(
             the ``Operation.grad_method`` attribute and the circuit structure will be analyzed
             to determine if the trainable parameters support the finite-difference method.
             If ``False``, the finite-difference method will be applied to all parameters.
-        shots (None, int, list[int]): The device shots that will be used to execute the tapes outputted by this
+        shots (None, int, list[int], ShotTuple): The device shots that will be used to execute the tapes outputted by this

```

Comment:
Do you mean `list[ShotTuple]` here instead of a single `ShotTuple`?

Reply:
right!